source/js/components/nsf-nav.js

```diff
@@ -32,9 +32,14 @@ var MenuLinks = React.createClass({
   },
   onScroll: function() {
     var links = this.state.links || [];
+    var hash = window.location.hash.replace(`#`, ``);
     var active = links[0] || ``;
     var scrollY = window.scrollY;
 
+    if (hash && !document.querySelector(`.nav-anchor#` + hash)) {
```

Comment:
I can see it now, thanks.

"If there are more than one element sharing the same id that means we are using it wrong" It's a little complex, but it's actually not a shared id. There are two sets of hashes that this component may end up processing. A valid one (From the nav) and an invalid one (From the accordion) If the id exists inside the element `.nav-anchor` it's a valid navigation id, and we can continue. If it doesn't exists in that element, the hash is invalid, either the accordion or something else, as you suggested `#test`.

We don't want to do anything in those cases. One top of that, if we do attempt to process the accordion hash, we end up overriding it and breaking accordion functionality.